packages/@aws-cdk/assert/lib/fs-utils.ts

```diff
@@ -0,0 +1,98 @@
+import fs = require('fs');
```

Comment:
I moved it to `assert` because I was using it both in `assets` and `aws-ecr-assets`.  I could move it to `assets` instead, and expose it to make to available to `aws-ecr-assets`?

Reply:
I see. Yes, I think put it in `assets`, and in fact you could import it directly from the `test/` directory in `aws-ecr-assets`:

```ts
import fsUtils = require('@aws-cdk/assets/test/fs-utils');
```

It's not super nice, but I don't want test helpers to be part of the public API of a package. This usage would be okay by me since it's between 1st-party packages only. Would be nice to spend a couple of comment lines on it when you're making that change.

Reply:
Done